src/internal/superjson/schema.ts

```diff
@@ -3,18 +3,16 @@ import {
   FILE_URI_REGEX,
   isFileURIString,
 } from '@superfaceai/ast';
-import { normalize } from 'path';
 
 export const trimFileURI = (path: string): string =>
-  normalize(path.replace(FILE_URI_REGEX, ''));
+  path.replace(FILE_URI_REGEX, '');
 
 export const composeFileURI = (path: string): string => {
   if (isFileURIString(path)) {
     return path;
   }
-  const normalizedPath = normalize(path);
 
-  return normalizedPath.startsWith('../')
-    ? `${FILE_URI_PROTOCOL}${normalizedPath}`
-    : `${FILE_URI_PROTOCOL}./${normalizedPath}`;
+  return path.startsWith('../')
+    ? `${FILE_URI_PROTOCOL}${path}`
+    : `${FILE_URI_PROTOCOL}./${path}`;
```

Comment:
question: since this doesn't normalize anymore (I assume because normalization now comes through the `IFilesystem`) shouldn't we normalize at the call sites? Is it possible to correctly emulate the original behavior (i.e. ignore already-prefixed paths) externally? I don't remember why exactly this was needed in the first place

Reply:
I did normalize at call sites

Reply:
How about these (and more)?

https://github.com/superfaceai/one-sdk-js/blob/cea3a9e80ec0165ce05fc9a27a47be50599051ec/src/internal/superjson/mutate.ts#L138

https://github.com/superfaceai/one-sdk-js/blob/cea3a9e80ec0165ce05fc9a27a47be50599051ec/src/internal/superjson/mutate.ts#L652

Reply:
🍆